### PR TITLE
Update rolling-upgrade paths

### PR DESCRIPTION
**Pull request by mfussenegger**
Master is already on 3.1 and contains streaming changes.

### CODE REVIEW

File: tests/bwc/test_rolling_upgrade.py [hunk 1]
```diff
@@ -14,7 +13,7 @@ def __repr__(self):
 
 
 ROLLING_UPGRADES = (
```
Comment (matriv) [inline tests/bwc/test_rolling_upgrade.py]: should we add also: `UpgradePath('3.1.x', 'latest-nightly')` ?

Comment (mfussenegger) [reply tests/bwc/test_rolling_upgrade.py]: Can't add it yet as we've no `3.1` releases yet

Comment (matriv) [reply tests/bwc/test_rolling_upgrade.py]: yeap sorry, stupid me

Comment (mfussenegger) [reply tests/bwc/test_rolling_upgrade.py]: I've had the same thought first :)

**Pull request merged**